### PR TITLE
feat: Tori colour palette updates

### PR DESCRIPTION
**Pull request by magnuh**
- Added new Tori blueberry color and updated values for watermelon
- Updated values for Tori red colors
- Updated Tori semantic colors to use blueberry instead of petroleum
- Updated values for watermelon in Tori semantic colors
- Updated Tori primary button tokens to use watermelon
- Updated token for Tori spinner to use blueberry instead of petroleum
- Updated Tori focused tokens to be blue

### CODE REVIEW

File: tokens/tori.fi/semantic.yml [hunk 1]
```diff
@@ -168,12 +168,12 @@ s:
...
         subtle: gray-50
-      link: petroleum-600
+      link: blueberry-600
       disabled: gray-300
       negative: red-600
       positive: green-600
       notification: white
 
-    focused: petroleum-400
+    focused: blueberry-400
```
Comment (BalbinaK) [inline tokens/tori.fi/semantic.yml:177]: Do we know if the focused color is the same in Figma (or wherever it needs to be :D ) so we don't need to update it later if it's not? I can't see any focused color in the [Taxonomy spreadsheet](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=1864511726).

**Review by BalbinaK — APPROVED** [review]
Looks great! What should we do before releasing 1.5.0? Test it locally with a designer to see if things look correct?

File: tokens/tori.fi/semantic.yml (hunk 1 above)
Comment (BalbinaK) [reply tokens/tori.fi/semantic.yml:177]: It looks like in the[ Figma overview ](https://www.figma.com/file/VcPPjSBp0sNMd5OuH1zVlJ/Tori-Colour-Concept?type=design&node-id=161-38448&mode=design&t=CFhHYoBqxsD62YU5-0)the outline might actually be `blue-300`.

Comment (magnuh) [reply tokens/tori.fi/semantic.yml]: Changed to `blue-300` 👍

**Comment by magnuh** [comment]
> Looks great! What should we do before releasing 1.5.0? Test it locally with a designer to see if things look correct?

Not sure. But I added the Release 1.6.0 PR. 🙂 https://github.com/warp-ds/css/pull/111

**Pull request merged**